demisto_sdk/commands/content_graph/strict_objects/script.py

```diff
@@ -0,0 +1,87 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import (
+    SKIP_PREPARE_SCRIPT_NAME,
+    TYPE_PYTHON2,
+    TYPE_PYTHON3,
+)
+from demisto_sdk.commands.common.StrEnum import StrEnum
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    Argument,
+    BaseIntegrationScript,
+    BaseStrictModel,
+    CommonFields,
+    Important,
+    Output,
+    ScriptType,
+)
+
+
+class SkipPrepare(StrEnum):
+    SCRIPT_NAME = SKIP_PREPARE_SCRIPT_NAME
+
+
+class ScriptSubType(StrEnum):
+    PYTHON3 = TYPE_PYTHON3
+    PYTHON2 = TYPE_PYTHON2
+
+
+class ScriptArgument(Argument):
+    description_x2: Optional[bool] = None
+
+
+class CommonFieldsScript(CommonFields):
+    id_x2: Optional[str] = None
+    id_xpanse: Optional[str] = Field(None, alias="id:xpanse")
+
+
+class ScriptImportant(Important):
+    description_x2: Optional[str] = None
+
+
+class EngineInfo(BaseStrictModel):
+    engine: Optional[str] = None
+
+
+class ContentItemFields(BaseStrictModel):
+    from_server_version: Optional[str] = Field(None, alias="fromServerVersion")
+
+
+class ContentItemExportableFields(BaseStrictModel):
+    content_item_fields: Optional[ContentItemFields] = Field(
+        None, alias="contentitemfields"
+    )
+
+
+class StrictScript(BaseIntegrationScript):
+    common_fields: CommonFieldsScript = Field(..., alias="commonfields")
+    name_x2: Optional[str] = None
+    script: str
+    type_: ScriptType = Field(..., alias="type")
+    tags: Optional[List[str]] = None
+    comment: Optional[str] = None
+    comment_marketplace_v2: Optional[str] = Field(None, alias="comment:marketplacev2")
+    enabled: Optional[bool] = None
+    args: Optional[List[ScriptArgument]] = None
+    script_target: Optional[int] = Field(None, alias="scripttarget")
+    timeout: Optional[str] = None
+    depends_on: dict = Field({}, alias="dependson")
+    depends_on_x2: dict = Field({}, alias="dependson_x2")
+    outputs: Optional[List[Output]] = None
+    important: Optional[List[ScriptImportant]] = None
+    docker_image: str = Field(None, alias="dockerimage")
```

Comment:
can we move to the parent?

Reply:
No, It is not shared between scripts and integrations.